### PR TITLE
chore: simplify openID flow and structure

### PR DESCRIPTION
**Pull request by redreceipt**
1. moves LandingSwiper inside of Auth
2. deletes redundant OpenID component
3. removes component prop overrides from LandingSwiper

Screen Shot 2022-09-09 at 3 22 49 PM

### CODE REVIEW

**Comment by redreceipt** [comment]
@vinnyjth this was work I did while debugging the openID bug. the fix for that is a one-liner and will come in another PR

**Comment by redreceipt** [comment]
@vinnyjth want to peek at this one? I'm going to do some other clean-up in onboarding on this one's shoulders trying to fix the birthday picker bug

**Comment by redreceipt** [comment]
Yea good call. I can do thatSent from my iPhoneOn Sep 21, 2022, at 3:57 PM, Vincent Wilson ***@***.***> wrote:﻿
@vinnyjth commented on this pull request.



In packages/apollos-ui-analytics/package.json:
> @@ -52,7 +52,7 @@
     "react-native-device-info": "^4.0.1"
   },
   "devDependencies": {
-    ***@***.***/client": "3.6.5",
+    ***@***.***/client": "^3.6.5",

why the fuzzy version?
I thought bad things happened if our apollo client versions ever got unsynchronized :)



In templates/mobile/src/auth/AuthLanding/AuthLandingConnected.js:
> @@ -36,7 +27,7 @@ const AuthLandingConnected = ({ provider = 'rock', navigation }) => {
   const { handleRequestOpenIdRegister } = React.useContext(LoginContext);
   const { login } = React.useContext(AuthContext);
 
-  const { data } = useQuery(OPENID_PROVIDERS_QUERY, {
+  const { data } = useQuery(GET_OPENID_PROVIDERS, {

👍



In yarn.lock:
> @@ -2,10 +2,10 @@
 # yarn lockfile v1
 
 
***@***.******@***.***":
-  version "3.6.5"
-  resolved ***@***.***/client/-/client-3.6.5.tgz#6a7baaf05852187b0eac1c6064e721d47326d70c"
-  integrity sha512-xGAZzv1f5abyH45MSU5eOLyGQuP4Ps0OhP36pSCtt6pCICI3n4yDkdftFCPvqDVqVTuciX1kkyeJPfGodz5kwg==
***@***.***/client@^3.6.5":

If we are going to use apollo client 3.6.9, why not bump the versions across all packages to 3.6.9 explicitly?
I don't mind the upgrade at all, but I do want to avoid apollo client version pain at all costs :)

File: templates/mobile/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@amplitude/react-native": "^2.12.1",
-    "@apollo/client": "3.6.5",
+    "@apollo/client": "^3.6.5",
```
Comment (redreceipt) [inline templates/mobile/package.json:33]:
```suggestion
    "@apollo/client": "3.6.9",
```

File: packages/apollos-ui-prayer/package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "yup": "0.26.6"
   },
   "devDependencies": {
-    "@apollo/client": "3.6.5",
+    "@apollo/client": "^3.6.5",
```
Comment (redreceipt) [inline packages/apollos-ui-prayer/package.json:67]:
```suggestion
    "@apollo/client": "3.6.9",
```

File: packages/apollos-ui-notifications/package.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
     "url": "^0.11.0"
   },
   "devDependencies": {
-    "@apollo/client": "3.6.5",
+    "@apollo/client": "^3.6.5",
```
Comment (redreceipt) [inline packages/apollos-ui-notifications/package.json:58]:
```suggestion
    "@apollo/client": "3.6.9",
```

File: packages/apollos-ui-fragments/package.json [hunk 4]
```diff
@@ -41,7 +41,7 @@
     ]
   },
   "dependencies": {
-    "@apollo/client": "3.6.5"
+    "@apollo/client": "^3.6.5"
```
Comment (redreceipt) [inline packages/apollos-ui-fragments/package.json:44]:
```suggestion
    "@apollo/client": "3.6.9"
```

File: packages/apollos-ui-analytics/package.json [hunk 5]
```diff
@@ -52,7 +52,7 @@
     "react-native-device-info": "^4.0.1"
   },
   "devDependencies": {
-    "@apollo/client": "3.6.5",
+    "@apollo/client": "^3.6.5",
```
Comment (redreceipt) [inline packages/apollos-ui-analytics/package.json:55]:
```suggestion
    "@apollo/client": "3.6.9",
```

File: yarn.lock [hunk 6]
```diff
@@ -2,10 +2,10 @@
 # yarn lockfile v1
 
 
-"@apollo/client@3.6.5":
-  version "3.6.5"
-  resolved "https://registry.yarnpkg.com/@apollo/client/-/client-3.6.5.tgz#6a7baaf05852187b0eac1c6064e721d47326d70c"
-  integrity sha512-xGAZzv1f5abyH45MSU5eOLyGQuP4Ps0OhP36pSCtt6pCICI3n4yDkdftFCPvqDVqVTuciX1kkyeJPfGodz5kwg==
+"@apollo/client@^3.6.5":
```
[the remark this answers is not in the document]
Comment (redreceipt) [reply yarn.lock]: done

**Review by vinnyjth — DISMISSED** [review]

**Comment by redreceipt** [comment]
@vinnyjth could you reapprove this one? something was funky so I dismissed your old one

**Review by camrun91 — APPROVED** [review]
Looks good to me!

**Pull request merged**